src/env.h

```diff
@@ -1015,6 +1015,9 @@ class Environment : public MemoryRetainer {
   uv_async_t task_queues_async_;
   int64_t task_queues_async_refs_ = 0;
 
+  std::atomic_bool is_stopping_{false};
```

Comment:
Consider adding a comment explaining why they must be initialized before async_hooks_.

Reply:
Good point, added.